vyper/evm/opcodes.py

```diff
@@ -209,7 +209,6 @@
     "NE": (None, 2, 1, 6),
     "DEBUGGER": (None, 0, 0, 0),
     "LABEL": (None, 1, 0, 1),
-    "GOTO": (None, 1, 0, 8),
```

Comment:
No longer necessary?

Reply:
No, it just doesn't exist as a pseudo opcode anymore because it can take varargs now. I added a special case for it in `LLLnode.py`.